data.json

```diff
@@ -11289,6 +11313,7 @@
             "root": "jockey",
             "symlinks": [
                 "applications-development",
+                "artsbuilder",
```

Comment:
From what I've found this is [Trinity's graphic design tool](https://centos.pkgs.org/7/trinity-r14-x86_64/trinity-artsbuilder-14.0.8-1.el7.x86_64.rpm.html) - why is it getting a spanner?